upgrading-from-v4.md

```diff
@@ -0,0 +1,66 @@
+# Upgrading from Docker Buildpack v4
+
+Docker Buildpack v5 contains some breaking changes and will require some changes in your CI/CD pipeline:
+
+* rootfs images are no longer published to Docker Hub.
+* rootfs images are now based on [Red Hat Universal Base Image 8 minimal](https://developers.redhat.com/articles/ubi-faq) image.
+
+## Building rootfs images
+
+In the past, Docker Buildpack offered two types of rootfs images: `bionic` and `ubi8`.
+
+* `bionic` images were based on Ubuntu 18.04 and were primarily used during the build phase - although it was also possible to build an app based on `bionic` by specifying a custom `--build-arg ROOTFS_IMAGE=mendix/rootfs:bionic` argument.
+* `ubi8` images were based on Red Hat UBI8 minimal and contained only components that are required to run a Mendix app.
+
+With Docker Buildpack v5, all images are now based on ubi8-minimal:
+
+* `app` images contain only components that are required to run an app. These images exclude any components that are only required during the build phase - reducing the image size and excluding components that could increase the number of unpatched CVEs in the final image
```

Comment:
Would changing this to just "Build-time components such as compilers, `tar` or Mono prerequisites are excluded - reducing ..." be a better option? If possible I'd like to avoid any sentences that sound redundant.